python/chronos/src/bigdl/chronos/benchmark/env_check.py

```diff
@@ -24,6 +24,7 @@
 
 from .utils_env import get_bytesize, _find_path, get_nano_env_var
 from bigdl.nano.common.common import _env_variable_is_set, _find_library
+from psutil import cpu_count
```

Comment:
If users only install bigdl-chronos[tensorflow], `psutil` module cannot be found. Maybe we need add `psutil` in setup.py later.

Reply:
Good catch, will add later.